src/pyscaffold/templates/setup_cfg.template

```diff
@@ -57,11 +57,15 @@ extras = True
 [tool:pytest]
 # Options for py.test:
 # Specify command line options as you would do when invoking py.test directly.
-# e.g. --cov-report html (or xml) for html/xml output or --junitxml junit.xml
-# in order to write a coverage file that can be read by Jenkins.
+# e.g. --junitxml junit.xml in order to write a coverage file that can be read
+# by Jenkins.
 addopts =
-    --cov ${root_pkg} --cov-report term-missing
+    --cov ${root_pkg}
+    --cov-config .coveragerc
+    --cov-report=
     --verbose
+# Please call `coverage combine .coverage` explicitly after running the tests.
```

Comment:
Is this only necessary when I use `tox` to run my tests or also in the case of `py.test`?

Reply:
Yes, you are right :sweat_smile:

When running with `py.test` directly, the package in the source code will be used directly (in this case we need to explicitly do `python setup.py develop` or `pip install -e .[testing,all]`), so the paths will be the right ones by default (no need for `coverage combine`).

(On the other hand, when trying to run `python setup.py test` without having installed pyscaffold in development/editable mode, I get an `ImportError: module 'pyscaffold.contrib.setuptools_scm.hg' has no attribute 'FILES_COMMAND'`)

Reply:
Since I cannot overwrite the same command line option `--cov-term` in `tox.ini` if they are specified in `setup.cfg`, my alternative proposal (a kind of trade-off that I am preparing another commit with it), would be leaving this option unspecified for `py.test`.

In this case, calling pytest directly would still print the coverage table but without the missing lines... Do you think this is a good compromise?

Reply:
Regarding the `ImportError: module 'pyscaffold.contrib.setuptools_scm.hg' has no attribute 'FILES_COMMAND'` problem, might it be that there is still an old PyScaffold 3.0.3 version around somehow? Do you also have the problem if you start with a fresh virtualenv and clean all `.eggs` as well as `.egg-info` directories in root and `src`? This problem is caused by the fact that the `entry_points` changed in the new setuptools_scm version.